shell/components/form/NameNsDescription.vue

```diff
@@ -220,39 +220,18 @@ export default {
       return this.nameDisabled || (this.mode === _EDIT && !this.nameEditable);
     },
 
-    namespaces() {
-      const currentStore = this.$store.getters['currentStore'](this.namespaceType);
-      const namespaces = this.namespacesOverride || this.$store.getters[`${ currentStore }/all`](this.namespaceType);
-      const filterNamespace = this.$store.getters['allNamespaces'];
-
-      const filtered = namespaces.filter( this.namespaceFilter || ((namespace) => {
-        // By default, include the namespace in the dropdown.
-        let out = true;
-
-        if (this.currentProduct?.customNamespaceFilter && this.currentProduct?.inStore) {
-          out = filterNamespace.find(NS => NS.metadata.name === namespace.metadata.name);
-        } else if (this.currentProduct?.hideSystemResources) {
```

Comment:
It is unclear if the logic related to fleet matches the one in the store, as they check different values while the condition itself is quite similar.

Reply:
I see the namespace store logic is updated to account for the user pref to toggle visibility of system namespaces but `hideSystemResources` should still be checked as it works independently

Reply:
But what is `hideSystemResources` for? in which case we have to check it?

Reply:
`hideSystemResources` allows a product to specify that system namespaces should be hidden _all_ the time - added for the sake of Harvester but could be used by any product potentially. I think it should be checked in the namespace store logic instead of every component that filters by ns as it currently is but either way it shouldn't be ignored in the NameNsDescription dropdown